mllib/src/test/scala/org/apache/spark/ml/classification/LinearSVCSuite.scala

```diff
@@ -272,36 +272,16 @@ class LinearSVCSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
 
     val coefficientsSK = Vectors.dense(7.24690165, 14.77029087, 21.99924004, 29.5575729)
     val interceptSK = 7.36947518
-    assert(model1.intercept ~== interceptSK relTol 1E-3)
-    assert(model1.coefficients ~== coefficientsSK relTol 4E-3)
+    assert(model.intercept ~== interceptSK relTol 1E-3)
+    assert(model.coefficients ~== coefficientsSK relTol 4E-3)
   }
 
-  test("linearSVC L-BFGS hinge comparison with R e1071 and scikit-learn") {
-    val trainer1 = new LinearSVC().setSolver(LinearSVC.LBFGS)
```

Comment:
we have to set RegParam to 0.00003 to get the similar result with sklearn, while the corresponding one should be 0.00002.